examples/defun.kl

```diff
@@ -37,4 +28,12 @@
 
 [example (last-stx '(a b c d e f g))]
 
+(defun map-syntax (f stx)
```

Comment:
this definition of `map-syntax` is dead code; should there be an `example`, or perhaps it should be in the `export` list?

Reply:
I should just delete it. I had an incorrect `flet*`, and that led to type errors in this, so I had it here so I could test more easily.